src/libstd/primitive_docs.rs

```diff
@@ -255,6 +255,11 @@ mod prim_unit { }
 ///
 /// *[See also the `std::ptr` module](ptr/index.html).*
 ///
+/// [`null`]: ../std/ptr/fn.null.html
```

Comment:
why go up and then go back into `std` again? that is why not

    ptr/fn.null.html